### DISCUSSION TITLE
Will this get merged into alpinejs?

### DISCUSSION THREAD

**Discussion by pfeatherstone** [alpine-collective/alpine-magic-helpers#143] [General]
Is this repository meant to be a placeholder for future features that will eventually make it into the main framework?
(A bit like opencv-contrib feading into opencv ?)

Or is this run independently in parallel?

**Comment by HugoDF** [comment]
It's run independently/by the community.

Most of the plugins are targeting Alpine V2 at the moment.

**Reply by KevinBatdorf** [reply]
We should add a note to the readme probably

**Reply by pfeatherstone** [reply]
Cool. Coz $get and $post will probs end up in the framework for example. So I wondered if this repo was like experimental features.

**Reply by HugoDF** [reply]
> $get and $post will probs end up in the framework for example

This is quite unlikely actually, Alpine is not a JS framework as much as it is a UI library centered around enhancing server-rendered applications with ease
